src/meta/model_v2/migration/src/m20240911_083152_variable_vnode_count.rs

```diff
@@ -0,0 +1,69 @@
+use sea_orm_migration::prelude::{Table as MigrationTable, *};
+
+const VNODE_COUNT: i32 = 256;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        manager
+            .alter_table(
+                MigrationTable::alter()
+                    .table(Table::Table)
+                    .add_column(
+                        ColumnDef::new(Table::VnodeCount)
+                            .integer()
+                            .default(VNODE_COUNT),
```

Comment:
```suggestion
                            .default(VNODE_COUNT)
                            .not_null(),
```

Nits, keep consistent with model definition since the column type is not `Option<i32>`.

Reply:
Thanks. Will address in following PRs.